flag/service/eventer/eventer.go

```diff
@@ -0,0 +1,10 @@
+package eventer
+
+import (
+	"github.com/giantswarm/draughtsman-operator/flag/service/eventer/github"
+)
+
+type Eventer struct {
```

Comment:
This is all the same as in the eventer or draughtsman itself. Exception is we only need the eventer type, the Github oauth token and the Github organisation.